src/main/java/classes/Storage.java

```diff
@@ -0,0 +1,119 @@
+package classes;
+
+import exceptions.DukeException;
+import exceptions.FolderNotFoundException;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+public class Storage {
```

Comment:
Perhaps it would be good to include JavaDoc header comments for this, as well as all other public classes and methods? This would help users better understand what each class and method is supposed to do.

Reply:
Sorry for any inconvenience caused as you were looking through. 
An updated version with JavaDoc headers incorporated has been pushed! Do take a look at them!

*Note: Classes under the folder "test" has yet gotten their JavaDocs as the test classes are still work in progress.